packages/client/src/components/RatingPopup/RatingPopup.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import './ratingPopup.scss';
+// @ts-ignore
```

Comment:
Last thing: Add "@types/react-star-rating-component": "^1.4.0" to client package.json and i think you can remove this ts-ignore

Reply:
Maybe we can disable this implicit any check on eslint Hahaha

Reply:
idk loll